core/src/main/java/org/infinispan/commands/write/PutForStateTransferCommand.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.commands.write;
+
+import org.infinispan.commands.CommandInvocationId;
+import org.infinispan.metadata.Metadata;
+
+public class PutForStateTransferCommand extends PutKeyValueCommand {
```

Comment:
I am thinking we should have this class should implement `LocalCommand` so we can easily see which commands aren't to be replicated.